unittests/util/MockRPCEngine.h

```diff
@@ -47,7 +47,7 @@ struct MockAsyncRPCEngine
     bool
     post(Fn&& func, std::string const& ip)
     {
-        boost::asio::spawn(ioc_, [f = std::move(func)](auto yield) { f(yield); });
+        boost::asio::spawn(ioc_, [f = std::move(func)](auto yield) mutable { f(yield); });
```

Comment:
I'm never a fan of names like `f` but in this case it actually works because it's in one line so you can easily see what `f` is. I'm ok to leave it but let's keep this in mind for future ⭐